modular_boh/code/modules/jobs/jobs.dm

```diff
@@ -406,3 +406,35 @@ var/const/INF               =(1<<11)
 			if(M.mind.assigned_role == "SolGov Representative")
 				to_chat(M, SPAN_NOTICE("<b>Your bodyguard, Agent [person.real_name], is present on [GLOB.using_map.full_name].</b>"))
 	..()
+
+/datum/job/liaison
+	access = list(
+		access_liaison, access_bridge, access_torch_fax, access_solgov_crew, access_keycard_auth,
+		access_nanotrasen, access_commissary,
+		access_radio_comm, access_radio_serv,
+		access_sec_doors, access_medical, access_research, access_cargo, access_engine, access_hangar
+	)
+
+/datum/job/bodyguard
+	access = list(
+		access_liaison, access_bridge, access_torch_fax, access_solgov_crew,
+		access_nanotrasen, access_commissary,
+		access_radio_comm, access_radio_serv,
+		access_sec_guard, access_security, access_sec_doors, access_medical, access_research, access_cargo, access_engine, access_hangar
```

Comment:
```suggestion
		access_security, access_sec_doors, access_medical, access_research, access_cargo, access_engine, access_hangar
```

Reply:
APAs already had access_sec_guard. I'll remove it but if people get pissed I'm sending them your way for messing up their accesses.